lib/active_record/acts_as/instance_methods.rb

```diff
@@ -9,6 +9,10 @@ def is_a?(klass)
         super || acting_as?(klass)
       end
 
+      def saved_changes?
```

Comment:
Wouldn't this method need to be called `has_changes_to_save?`?

Reply:
This should be called `saved_changes?` as it is extending the `acts_as` model's `saved_changes?`. It is called in the `after_update` callback of the `acts_as` model [here](https://github.com/krautcomputing/active_record-acts_as/blob/master/lib/active_record/acts_as/relation.rb#L31). At this point, the `acts_as` model has been saved, while the `acting_as` model has not yet been saved yet.

Reply:
OK!